request.c

```diff
@@ -482,8 +479,18 @@ void getTopClients(char *client_message, int *sock)
 		temparray[i][1] = clients[i].count;
 	}
 
+	// Sort in descending order?
+	bool desc = false;
+	if(command(client_message, " desc"))
+	{
+		desc = true;
+	}
+
 	// Sort temporary array
-	qsort(temparray, counters.clients, sizeof(int[2]), cmpasc);
```

Comment:
Added optional `desc` sorting here, this is basically a copy of the code in lines 369-372.